main.css

```diff
@@ -442,3 +449,86 @@ button:hover {
 .proj-cad {
   display: grid;
 }
+
+.contact-me {
+  background-color: #6070ff;
+  border: 1px solid #cfd8dc;
+  border-radius: 100px 0 0;
+  width: 372px;
+}
+
+.bg-image {
+  background-image: url(images/shapes.png);
+  background-repeat: no-repeat;
+  background-position-x: right;
+}
+
+.contact-title {
+  font-family: "Poppins", sans-serif;
+  font-style: normal;
+  font-weight: bold;
+  font-size: 40px;
+  line-height: 52px;
+  display: flex;
+  align-items: center;
+  text-align: center;
+  color: #fff;
+  height: 56px;
+  justify-content: center;
+  padding-top: 80px;
+}
+
+.contact-text {
+  padding-left: 24px;
+  display: flex;
+  align-items: center;
+  color: #fff;
+  justify-content: center;
+  width: 320.5px;
+  height: 146px;
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 16px;
+  line-height: 24px;
+  margin: 0.5px;
+}
```

Comment:
- [ ] Please, consider Figma design when you style. Here is the requirements from Figma regarding how your "contact text" should display:
<img width="228" alt="Screen Shot 2022-01-11 at 22 23 03" src="https://user-images.githubusercontent.com/49064106/148999896-e9b4f43d-e1ad-4f15-9b5d-e55c01f026bf.png">
Please, consider the font-size, line-height and etc.